x-pack/plugins/reporting/server/config/create_config.ts

```diff
@@ -44,7 +44,7 @@ export function createConfig(
     ipaddr.isValid(kibanaServerHostname) &&
     !sum(ipaddr.parse(kibanaServerHostname).toByteArray())
   ) {
-    logger.warn(
+    logger.info(
       `Found 'server.host: "0.0.0.0"' in Kibana configuration. Reporting is not able to use this as the Kibana server hostname.` +
         ` To enable PNG/PDF Reporting to work, 'xpack.reporting.kibanaServer.hostname: localhost' is automatically set in the configuration.` +
         ` You can prevent this message by adding 'xpack.reporting.kibanaServer.hostname: localhost' in kibana.yml.`
```

Comment:
Oops, I think a little too much got reverted, and we lost the "softening" of the log message:

```suggestion
      `Overriding server host address "0.0.0.0" in Reporting runtime config,` +
        ` using "xpack.reporting.kibanaServer.hostname: localhost".`
```